### PR TITLE
update intervale documetation with covid reopening rules

### PR DESCRIPTION
**Pull request by PhiloEpisteme**
This PR updates the Intervale documentation with COVID-19 rules. The rules themselves come from coordinating with the Karina, the president of MITOC, and the Office of General Counsel and SOLE.

### CODE REVIEW

**Comment by PhiloEpisteme** [comment]
@fpagnoux I'll update the PR with the correct URLs once I get them from Karina. I'll also hold off on merging until Karina gives me the thumbs up on the content from a rules perspective.

File: _pages/rentals/intervale-manual.md [hunk 1]
```diff
@@ -4,11 +4,39 @@
...
+
+- **Vaccination Status:** All guests must use MIT's Tim Tickets "Visitor" portal to attest that they are either fully vaccinated or that they have a valid medical or religious excemption.
+- **Daily Attestation:** All guests staying at the property must use MIT's Tim Tickets "Visitor" portal to submit a daily attestation of health.
+- **Guest Information:** For the purposes of contact tracing, the full name of each guest as well as their contact information (email or phone) must be provided at the time of reservation.
+
+### <span style="color:red;">Cabin & Yurt</span>
+
+In addition to all of the requirements above for camping & outdoor use only, the following rules also apply to using the cabin or yurt.
+
+- **Occupancy:** The cabin and yurt are indoor spaces and may be reserves separately. Only 1 group of up to 10 people may reserve the cabin or yurt for any single night.
```
Comment (fpagnoux) [inline _pages/rentals/intervale-manual.md:24]:
```suggestion
- **Occupancy:** The cabin and yurt are indoor spaces and may be reserved separately. Only 1 group of up to 10 people may reserve the cabin or yurt for any single night. 
```

File: _pages/rentals/intervale-manual.md [hunk 2]
```diff
@@ -4,11 +4,39 @@
...
 
-## Purpose
+# <span style="color:red;">Use During Covid</span>
+
+In order to mitigate the risks of COVID-19, some rules for accessing Intervale
+have been modifed. Below is a list of all updated rules pertaining to Intervale
+access and COVID-19. These rules are intended to supercede any other rule
+listed on this page which may be in contradiction.
+
+### <span style="color:red;">Outdoor/Cambing Only</span>
```
Comment (fpagnoux) [inline _pages/rentals/intervale-manual.md:14]:
```suggestion
### <span style="color:red;">Outdoor/Camping Only</span>
```

**Review by fpagnoux — APPROVED** [review]
Looks good, good to merge once the URL is fixed and the Karina approves

**Comment by PhiloEpisteme** [comment]
@fpagnoux can you, @DavidCain or @mprat would one of you mind merging this PR. Karina has given her thumbs up.

**Pull request merged**